src/renderer/pages/Assets/Assets/Assets.tsx

```diff
@@ -30,8 +33,14 @@ export const Assets = () => {
           </div>
         </Header>
         <ShardSelectorButton />
-
-        <AssetsChainView query={query} activeShards={activeShards} hideZeroBalances={hideZeroBalances} />
+        <div className="flex flex-col gap-y-4 w-full h-full overflow-y-scroll">
+          {activeShards.length > 0 &&
+            (assetsView === AssetsListView.CHAIN_CENTRIC ? (
+              <AssetsChainView query={query} activeShards={activeShards} hideZeroBalances={hideZeroBalances} />
+            ) : (
+              <AssetsPortfolio />
+            ))}
```

Comment:
Assets page should only include features, features must decide wheather to render or not
`AssetsPortfolio` & `AssetsChainView` should check `assetsView === AssetsListView.CHAIN_CENTRIC`
and return `null` or meaningful content.

Reply:
I think in general it's more common to have a ternary operator for such cases so it could be easier for outsiders and  newcomers to understand what's going on here

Reply:
@tuul-wq I tried this way but in this case, on the `Assets` page we have a structure like this: 

```
 <AssetsChainView query={query} activeShards={activeShards} hideZeroBalances={hideZeroBalances} />
 <AssetsPortfolio />
```

And it's not obvious that it is not just part of the page's component but a separate view of this page when one excluded another. If we have it this way, then we can't read the page at first glance, we need to go deeper through files to understand what's going on there and how this works

Reply:
That's totally fine, because page includes features/widgets so each UI component leads us directly to dedicated functionality
We could change Assets views names so they more clearly tell us about their purpose
`AssetsChainView` - `AssetsPortfolioView` 
This way anyone who's looking at the Assets page will see that it has X features (filter, settings, and a bunch of different views)